### DISCUSSION TITLE
Additional test suites not displayed in new UI

### DISCUSSION THREAD

**Discussion by JohannaOtt** [INSPIRE-MIF/helpdesk-validator#894] [Ideas]
Hi,

We implemented additional test suites for the validator. When using the old UI, they appear as an additional section: 
![grafik](https://user-images.githubusercontent.com/34163838/122343544-19a77f80-cf46-11eb-8e96-5b4174a40855.png)

I expected a similar behavior for the new UI, but when using it, the additional test is not offered any longer:
![grafik](https://user-images.githubusercontent.com/34163838/122349495-9b021080-cf4c-11eb-919c-3fe2c35edf29.png)

Is there any way to include custom test suites in the validator using the new UI?

All the best

Johanna

**Comment by fabianospinelli** [comment]
Currently the "custom tests" functionality has not been provided. Probably in the future this functionality will be implemented by adding configuration files for each individual custom test, but all of this is still being analyzed and it is impossible for us to give any timing.
